distribution/openrct2.d.ts

```diff
@@ -1206,7 +1206,7 @@ declare global {
     interface StaffHireArgs extends GameActionArgs {
         autoPosition: boolean;
         staffType: number; // 0: handyman, 1: mechanic, 2: security, 3: entertainer
-        entertainerType: number; // see EntertainerCostume in openrct2/entity/Staff.h
+        costumeIndex: number; // peep animation object id to use as costume
```

Comment:
This number is only used for entertainers, and currently not used for any other staff hires. Should we mark it as optional in this TypeScript struct? (The game will currently error if you omit it for entertainers.)

Reply:
While it would be nice, I think it's best to only mark as optional once the game doesn't throw errors anymore for omitting it.

AFAIK currently there's no other game actions with optional or fallback parameters either.